EASY_DOWNLOAD.m

```diff
@@ -1,13 +1,14 @@
 %% EASY DOWNLOAD
 
-system('git clone git://github.com/sharpee/mid.git mid', '-echo');
+origin = pwd;
```

Comment:
Might be even better to set this to `origin = fileparts(mfilename('fullpath'));` so then mid will be downloaded into "matlab_mid_wrapper" even if "EASY_DOWNLOAD.m" is executed from outside.